site/content/docs/team/aryan.md

```diff
@@ -0,0 +1,23 @@
+---
+handle: aryangupta701
+name: "Aryan Gupta"
+roles: Extended Team
+author: aryangupta701
+type: team
+github: aryangupta701
+linkedin: aryan-gupta-78273a1b6
+weight: 11
+---
+Aryan Gupta is a highly motivated undergraduate student pursuing a degree in Computer Science Engineering.
+
+Aryan is an enthusiastic advocate for open-source development, actively contributing to multiple projects in the open-source community. With a profound passion for software development, he possesses a strong aptitude for creative problem-solving. 
+
+His expertise spans a wide range of programming languages, including Java, JavaScript, TypeScript, C++, and Python.
+
+Aryan's proficiency as a full-stack developer, particularly in the MERN stack, reflects his commitment to continuous learning and staying at the forefront of technology.
+
+His inquisitiveness extends to the realm of cybersecurity, where he thrives on the challenge of identifying and mitigating online vulnerabilities.
+
+He was the Google Summer of Code 2023 student contributor for the [Browser Recorder](/blog/2023-09-11-browser-recorder/) feature for ZAP. 
+
+Contact Him Here: [GitHub](https://www.github.com/aryangupta701) | [LinkedIn](https://www.linkedin.com/in/aryan-gupta-78273a1b6/) | [Twitter](https://twitter.com/aryangupta_701)
```

Comment:
Dont really need these as they are already in the left panel?
Otherwise all good 😁

Reply:
oh yes right